### ISSUE TITLE
Faveo Helpdesk Docker on PHP7?

### ISSUE THREAD

**Issue by UnderDogg** [ladybirdweb/faveo-docker#2]
Just was taking a look at installing faveo on docker. It's using the dockerfile from this repo.
Unfortunately it's installing php5 on nginx? Any plans for updating that to php7?

**Comment by mverma16** [comment]
Yes we do have plans for updating to PHP7. But as our current Faveo stable version is on Laravel 5.2 and compatible with only PHP5.6. We have updated Laravel framework version to the latest and it is compatible with php7.1.3 and greater (which is Laravel 5.6's preferred version)
